Gemfile

```diff
@@ -5,11 +5,13 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.7.0'
 
-gem 'codebreaker_manflyy', '~> 0.2.0'
+gem 'codebreker_manfly', '~> 0.1.2'
```

Comment:
Why you change to previous version? And I dont see you game repo in list of your repositories

Reply:
I tried to rewrite, as you said.

Reply:
There was an unfinished version and I launched the latest one, which was successful

Reply:
And where is it? I dont see your codebreaker_game repository